dvc/command/config.py

```diff
@@ -92,6 +114,13 @@ def add_parser(subparsers, parent_parser):
         action="store_true",
         help="Unset option.",
     )
-    config_parser.add_argument("name", help="Option name.")
+    config_parser.add_argument("name", nargs="?", help="Option name.")
     config_parser.add_argument("value", nargs="?", help="Option value.")
+    config_parser.add_argument(
+        "-l",
+        "--list",
+        default=False,
+        action="store_true",
+        help="list all defined config values",
```

Comment:
```suggestion
        help="List all defined config values",
```

Reply:
@isidentical ^ looks like we've missed this :slightly_frowning_face: Mind submitting a PR to fix it?

Reply:
Amended here #5087

Reply:
Woot woot. Thanks guys!